mobile/src/navigations/root/libs/constants.ts

```diff
@@ -0,0 +1,38 @@
+import { RootScreenName } from '#libs/enums/enums';
+import { type NavigationItem } from '#libs/types/types';
+import { Main } from '#navigations/navigations';
+import { Auth } from '#screens/auth/auth';
+import { Survey } from '#screens/survey/survey';
+
+const NAVIGATION_ITEMS: NavigationItem[] = [
+  {
+    name: RootScreenName.MAIN,
+    component: Main,
+    conditionToRender: (authenticatedUser, isSurveyCompleted): boolean => {
```

Comment:
let's rename it to checkShouldBeRendered according to quality criteria (functions should be named as actions)
```suggestion
    checkShouldBeRendered: (authenticatedUser, isSurveyCompleted): boolean => {
```